README.md

```diff
@@ -384,3 +386,48 @@ const assert = v.assert(v.range([-10, 10]));
 assert(4); // pass
 assert(-100); // fail
 ```
+
+
+## Custom validators
+One of the primary goals of Fusspot is to be customizable out of the box. There are multiple ways to which one can create a custom validator. 
```

Comment:
Can we explicitly state how you *use* the custom validator function that you've written?

I believe that you can pass your custom validators into `v.assert` just like any of the built-in validators — right?

Reply:
Yup, I tried to add a bit more details, let me know if it works.